app/views/orders/index.html.erb

```diff
@@ -0,0 +1,29 @@
+<p id="notice"><%= notice %></p>
+
+<h1>Orders</h1>
+
+<table>
+ <thead>
+  <tr>
+    <th>Product name</th>
+    <th>Product count</th>
+    <th colspan="2"></th>
+  </tr>
+ </thead>
+
+ <tbody>
+  <% @orders.each do |order| %>
+    <tr>
+      <td><%= order.product_name %></td>
+      <td><%= order.product_count %></td>
+      <td><%= link_to 'Show', order %></td>
+      <td><%= link_to 'Edit', edit_order_path(order) %></td>
+      <td><%= link_to 'Destroy', order, method: :delete, data: {confirm: 'Are you sure?'} %></td>
+    </tr>
+  <% end %>
+ </tbody>
+</table>
+
+<br>
```

Comment:
br tags are not necessarily bad, but generally, it is better to avoid them. We can use css to to add margin to mimic br tag functionality.